test/hotspot/jtreg/runtime/cds/appcds/VerifyWithDefaultArchive.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2021, Alibaba Group Holding Limited. All Rights Reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ *
+ */
+
+/*
+ * @test 8264337
+ * @summary test default cds archive when turning on VerifySharedSpaces
+ * @requires vm.cds
+ * @library /test/lib
+ * @run driver VerifyWithDefaultArchive
+ */
+
+import jdk.test.lib.process.ProcessTools;
+import jdk.test.lib.process.OutputAnalyzer;
+
+public class VerifyWithDefaultArchive {
+    public static void main(String... args) throws Exception {
+        ProcessBuilder pb = ProcessTools.createJavaProcessBuilder("-Xlog:cds", "-XX:+VerifySharedSpaces", "-version");
+        OutputAnalyzer out = new OutputAnalyzer(pb.start());
+        out.shouldContain("OpenJDK");
+        out.shouldHaveExitValue(0);
+    }
+}
```

Comment:
Please add the newline at the end of file here. See how UI complains about it :)

Reply:
Got it. But I'm not sure what's the abbreviation `UI` means... Could you please give me a link for that? Thanks!

Reply:
Thanks! I meant how GitHub UI (user interface) shows the red stop sign in cases like these.